js/controller.js

```diff
@@ -2205,7 +2216,7 @@ OO.plugin('Html5Skin', function(OO, _, $, W) {
       }
     },
 
-    toggleScreen: function(screen) {
+    toggleScreen: function(screen, doNotPause) {
```

Comment:
Can you add a jsdoc since you're modifying the function signature?

Reply:
Haha deja vu, will add